### DISCUSSION TITLE
Client configuration for elasticache redis cluster

### DISCUSSION THREAD

**Discussion by fzn16** [redis/go-redis#2045] [Q&A]
Configuring a elasticache redis cluster with cluster mode enabled(multiple shards) gives you a cluster configuration endpoint.

Question is when configuring redis.NewClusterClient do we provide cluster configuration endpoint that elasticache provides or do we provide a list of all the nodes that are part of the cluster?

**Comment by bibicu123** [comment]
I would like to know this myself !
I am having trouble using AWS with cluster enabled

**Comment by ndyakov** [comment]
@fzn16 @bibicu123 I assume you found the answer to your questions, but to summarize it briefly you can use a single address and the client should be able to fetch the cluster topology.
